pii_recognizer/item.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+categories: 
+  - machine-learning
+  - data-preparation
+description: This function is used to recognize PII in a given text
+doc: ''
+example: pii_recognizer.ipynb
+generationDate: 2023-07-30:21-01
+icon: ''
+labels: {}
+maintainers: []
+marketplaceType: ''
+mlrunVersion: ''
+name: pii-recognizer
+platformVersion: ''
+spec:
+  filename: pii_recognizer.py
+  handler: recognize_pii
+  image: mlrun/mlrun
+  kind: job
+  requirements:
+   - mlrun==1.4.0-rc19
```

Comment:
The requirements are "add-ons" to the base image (here - `mlrun/mlrun`) so you are not suppose to install mlrun on your image.
Also it is a better practice to use a stable version of mlrun, and set in the `mlrunVersion` field your desired version (I guess you wish to update it to `1.4.0`)